setup.py

```diff
@@ -59,6 +59,7 @@
                         'psutil>=3.3.0',
                         'intervaltree',
                         'sortedcontainers<2.0', # See Gallopsled/pwntools#1154
+                        'sphinx==1.6.7',
```

Comment:
Sphinx is not required for pwntools itself, only testing.  See `docs/requirements.txt`.

Separately, we don't want to pin to any particular version, this is very non-Pythonic.

Reply:
I just found that someone else (#1199 ) had the same issue. I'd assume because the TESTING.md doesn't mention `pip2 install -r docs/requirements.txt` anywhere and the setup-scripts don't do it either.

Reply:
Well, anyway, we do not want pwntools to automatically install unnecessary dependencies.
```suggestion
```